core/pom.xml

```diff
@@ -487,9 +487,9 @@ THE SOFTWARE.
       <version>20110809</version>
     </dependency>
     <dependency>
-      <groupId>commons-collections</groupId>
-      <artifactId>commons-collections</artifactId>
-      <version>3.2.1</version>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-collections4</artifactId>
```

Comment:
I am adding version 4 alongside version 3 - a bunch of the test failures from the first time around seemed to be from plugins, as predicted.